src/locale/en.json

```diff
@@ -1,4 +1,14 @@
 {
+  "homepage": {
+    "welcome": "Welcome to Databus",
+    "databus_definition": "The open platform for real data on the quality of public transportation lines in Israel",
+    "website_goal": "The purpose of the site is to improve the quality of public transportation in Israel by providing reliable information to journalists, citizens, transportation companies, and government officials in Israel.",
+    "timeline": "Schedule history",
+    "gaps": "Rides that didn't take place",
+    "gaps_patterns": "Patterns",
+    "map": "Time based map",
```

Comment:
we can use same translation keys from the menu, so changing one place will also change the other

Reply:
Done